static/searchPage.html

```diff
@@ -43,6 +44,10 @@ <h2 id="session-name">Session name</h2>
           placeholder="Enter a place, town, or postal code:"
         />
       </div>
+      <p id="input-instructions">
+        Please do not use special characters other than the following: [ . ] [ ,
+        ] [ _ ] [ - ] and Space.
+      </p>
```

Comment:
Something to consider: Would it be better to show this as an alert error or something? This is absolutely fine for now tho.

Reply:
What this message actually says is something we can discuss, but I think it makes sense to give some instruction to the user about which characters not to use. That way the user knows before they submit any information. It might be frustrating to try to submit something, only to get an error to try it again without certain characters rather than saying outright which ones are not allowed.
 
Also, there already is an alert error that shows up if they submit special characters despite the warning.

Reply:
What this message actually says is something we can discuss, but I think it makes sense to give some instruction to the user about which characters not to use. That way the user knows before they submit any information. It might be frustrating to try to submit something, only to get an error to try it again without certain characters rather than saying outright which ones are not allowed.
 
Also, there already is an alert error that shows up if they submit special characters despite the warning.

Reply:
I ended up removing this. Now all special characters are encoded when making a geocode api request.